site/docs/uk/README.md

```diff
@@ -0,0 +1,103 @@
+---
+home: true
+heroImage: /images/Y.png
+actions:
+  - text: Розпочати
+    link: /uk/guide/getting-started.html
+    type: primary
+  - text: Вступ
+    link: /uk/guide/introduction.html
+    type: secondary
+features:
+  - title: Простий у використанні
+    details: grammY робить створення ботів Telegram настільки простим, що ви вже знаєте, як це зробити.
+  - title: Гнучкий
+    details: grammY відкритий і може бути розширений за допомогою плагінів, щоб точно відповідати вашим потребам.
+  - title: Масштабований
+    details: grammY допоможе вам, коли ваш бот стане популярним і трафік зросте.
+permalink: /uk/
+---
+
+<h6 align="right">… {{ [
+  'подумайте, чомУ',
+  'нова ера розробки ботів',
+  'швидший за вас',
+  'попереду ще одне оновлення',
+  'може зробити все, окрім вечері',
+  'легко, як з обійстя виховати козУ',
+  'обслуговано сотні мільйонів',
+][Math.floor(Math.random() * 7)] }}.</h6>
+
+## Швидкий старт
+
+Боти, написані на [TypeScript](https://www.typescriptlang.org) або JavaScript, працюють на різних платформах, зокрема [Node.js](https://nodejs.org).
+
+`npm install grammy` і вставте наступний код:
+
+<CodeGroup>
+  <CodeGroupItem title="TypeScript" active>
+
+```ts
+import { Bot } from "grammy";
+
+const bot = new Bot(""); // <-- Помістіть токен свого бота (https://t.me/BotFather)
```

Comment:
Just want to make sure that this is the translation of `put your bot token between the ""`, right? Because I don't see any quotation marks in this translation.

Reply:
Indeed, the quotation marks went somewhere. Fixed in a separate commit.